product_subscription/models/invoice.py

```diff
@@ -56,16 +81,17 @@ def confirm_paid(self):
                                   ('origin', '=', invoice.move_name)])
 
             if invoice.subscription and invoice.type == 'out_invoice' and not refund:
-                effective_date = datetime.now().strftime('%Y-%m-%d')
-                # effective_date = datetime.now().strftime('%d/%m/%Y')  fixme was this, why?
+                # seems that doing it this way strftime('%Y-%m-%d') doesn't
+                # work
+                effective_date = datetime.now().strftime('%d/%m/%Y')
 
                 # take the effective date from the payment.
                 # by default the confirmation date is the payment date
                 if invoice.payment_move_line_ids:
                     move_line = invoice.payment_move_line_ids[0]
                     effective_date = move_line.date
 
-                invoice.process_subscription(effective_date)
```

Comment:
@houssine this goes through post_process_confirm_paid defined in `easy_my_coop.models.account_invoice.py` it therefore calls set_cooperator_effective and creates a cooperator share line at line 78

Reply:
I see... in fact it should go back to invoice.process_subscription(effective_date) as process_subscription function is not defined in easy my coop. Did I did this ?

Reply:
I think you did this yes (_git blame is watching you_) :p The other option is to set the invoice as `cooperator_share = True` like you do for subscriptions. Can you fix it today or should I do it?

Reply:
ok done. I finally changed the function name as these functions don't rely on each other. this is the confirm paid that handle the process and hence should call the right function